### PR TITLE
Use logging instead of _msg()

### PR DESCRIPTION
**Pull request by nafur**
Use standard logging facilities.

### CODE REVIEW

File: ddsmt.py [hunk 1]
```diff
@@ -215,10 +213,9 @@ def _process_substitutions(pool, exprs, superset, superset_substs):
                     restart = True
                     break
 
-                _msg(2,
```
Comment (mpreiner) [inline ddsmt.py:218]: Is there a way to have the same behavior as before with logging? This updates the current line instead of printing a new one.

Comment (nafur) [reply ddsmt.py:218]: Hm, not easily. Maybe we can just use `sys.stdout.write()` manually at this one point. It is kind of special anyway...

Comment (mpreiner) [reply ddsmt.py:219]: Yes, sounds good, it's the only place we need it.

File: ddsmt.py [hunk 2]
```diff
@@ -214,9 +214,11 @@ def _process_substitutions(pool, exprs, superset, superset_substs):
                     restart = True
                     break
 
-                logging.debug("granularity: {}, subset {} of {}, "\
-                      "s-expressions: {}/{}".format(gran, i, len(subsets),
-                          nexprs - nreduced_total, nexprs))
+                if options.args().verbosity >= 2:
+                    sys.stdout.write(
```
Comment (mpreiner) [inline ddsmt.py:218]: Maybe also add the ddSMT prefix.

**Pull request merged**